geode-core/src/main/java/org/apache/geode/internal/cache/wan/AbstractGatewaySender.java

```diff
@@ -180,6 +180,9 @@
 
   protected volatile ConcurrentLinkedQueue<EntryEventImpl> tmpDroppedEvents =
       new ConcurrentLinkedQueue<>();
+
+  private volatile boolean allInstancesStopped = false;
```

Comment:
This name seems not correct. What you really enable/disable is actually the tmpDroppedEvents. 

I suggest to use disableTmpDroppedEvents.